### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Victoria-Moiseienko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-"# project24" 
+
+##internet-shop
+Internet-shop with basic functions. 
+According to N-tier architecture project contain several layers:
```
Comment (AnaLanda) [inline README.md]:
```suggestion
In accordance with the N-tier architecture principles, this project is comprised of several layers:
```

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+* Manage products
+* Manage users
+* View and delete orders
+###for User:
+* Registration
+* Login/Logout
+* Buy products
+* View Shopping Cart
+* Create orders
+* View own orders and its details
```
Comment (AnaLanda) [inline README.md]:
```suggestion
###for the Admin: 
* Managing products
* Managing users
* Viewing and deleting orders
###for the User:
* Registration
* Login/Logout
* Buying products
* Viewing the shopping cart
* Completing orders
* Viewing own orders and their details
```

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+##technologies 
+Java, Servlets, RBAC Filters, Tomcat, JSP, JSTL, JDBC, MySQL
```
Comment (AnaLanda) [inline README.md]: I think you could add the version of Java you used (e.g. Java 8, Java 11) and format this list as bullet points, e.g.
- Java 8
- Servlets

File: README.md [hunk 4]
```diff
@@ -1 +1,29 @@
...
+ 
+##to run project
+1. Set up MySql DB and Workbench
+2. Run the sql-code from init_db.sql - placed in 'resources' package
+3. Use your properties in ConnectionUtil class
+4. Configure Tomcat
```
Comment (AnaLanda) [inline README.md]:
```suggestion
##to run the project
1. Set up MySql DB and Workbench on your machine
2. Run the SQL-code from the _init_db.sql_ file located in the _resources_ package
3. Insert your MySQL Workbench login and password in the properties in the ConnectionUtil class
4. Configure Tomcat
```
I'd also mention the need to 'inject data' when the website is up and running and how one can try the website out as a user and an admin (e.g. you could say which login and password you've set for the admin)

**Review by AnaLanda — COMMENTED** [review]
Looks good overall)))

File: README.md (hunk 1 above)
Comment (Victoria-Moiseienko) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Victoria-Moiseienko) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Victoria-Moiseienko) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (Victoria-Moiseienko) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -1 +1,39 @@
-"# project24" 
+
+##internet-shop
```
Comment (Sofasmile) [inline README.md]:
```suggestion
## internet-shop
```

File: README.md [hunk 6]
```diff
@@ -1 +1,39 @@
...
+* DAO
+* Services
+* Controllers
+* Views
+##main functions
+###for the Admin: 
+(login: admin, password: admin)
+* Managing products
+* Managing users
+* Viewing and deleting orders
```
Comment (Sofasmile) [inline README.md]:
```suggestion
* Views

## main functions

### for the Admin: 
(login: admin, password: admin)
* Managing products
* Managing users
* Viewing and deleting orders
```
зараз це виглядає так:
image

**Review by Sofasmile — COMMENTED** [review]
видали імпл на Storage

File: README.md (hunk 6 above)
Comment (Victoria-Moiseienko) [reply README.md]: Done

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**